server/json-schemas/noncommercial-application-info-schema.es6

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+  'id': '/noncommercialApplicantInfo',
+  'type': 'object',
+  'allOf':[
+    {'$ref':'/applicantInfoBase'},
+    {
+      'properties': {
+        'organizationName': {
+          'default':'',
+          'type': 'string'
+        },
+        'orgType':{
+          'default':'',
+          'description':'Organization Type',
+          'enum':[
```

Comment:
This orgType enum doesn't match the middle layer. I think it's docs and schema both offer the latest list. :)